pyproject.toml

```
@@ -16,8 +16,6 @@ builtins = [
 	"npgettext",
 ]
 
-logger-objects = ["logHandler.log"]
-
 include = [
 	"*.py",
 	"*.pyw",
```

Add the time of flake8, we also linted scons scripts.

```suggestion
	"*.pyw",
	"sconstruct",
	"*sconscript",
```

This needs testing though, probably also a fixup commit to prelint these